src/Routing/UrlGenerator.php

```diff
@@ -324,18 +345,11 @@ protected function formatParametersForUrl($parameters)
      *
      * @param  array  $parameters
      * @return array
+     * @deprecated v5.8.0
      */
     protected function replaceRoutableParametersForUrl($parameters = [])
     {
-        $parameters = is_array($parameters) ? $parameters : [$parameters];
-
-        foreach ($parameters as $key => $parameter) {
-            if ($parameter instanceof UrlRoutable) {
-                $parameters[$key] = $parameter->getRouteKey();
-            }
-        }
-
-        return $parameters;
+        return $this->formatParameters($parameters);
```

Comment:
Both `formatParametersForUrl()` and `replaceRoutableParametersForUrl()` is only used internally, but I'm not sure how Lumen deprecation structure works on I just mark it for deprecated at the moment.

Reply:
Imo can be removed since it's for a new release.

Reply:
Done